nordic-nrf51822-16k-armcc/CMake/toolchain.cmake

```diff
@@ -67,6 +68,9 @@ function(yotta_apply_target_rules target_type target_name)
             # and append the softdevice hex file
             COMMAND python ${NRF51822_MERGE_HEX_SCRIPT} ${NRF51822_SOFTDEVICE_HEX_FILE} ${target_name}.hex ${target_name}-combined.hex
             COMMENT "hexifying and adding softdevice to ${target_name}"
+            # printing memory usage information
+            COMMAND python ${NRF51822_MEMORY_INFO_SCRIPT} ${target_name}
+            COMMENT "printing memory usage information"
```

Comment:
I'm not sure cmake COMMENT works by annotating the preceding COMMAND. COMMENT might be supplying a comment to an entire block of add_custom_command().  Perhaps you should create a separate add_custom_command() block for printing out the size. You should also remove the invocation of arm-none-eabi-size (from above).